sites/website/src/css/custom.css

```diff
@@ -24,10 +24,6 @@
   padding: 0 var(--ifm-pre-padding);
 }
 
-div.react-toggle {
-  display: none;
-}
```

Comment:
🤣

Reply:
A fine bit of code that is sadly no-longer needed ;)